daphne/src/testing.rs

```diff
@@ -444,6 +444,10 @@ where
 
         Ok(())
     }
+
+    async fn current_batch(&self, _task_id: &Id) -> std::result::Result<Id, DapError> {
+        panic!("current_batch not implemented yet");
```

Comment:
In fact, we already implement something like this: search `current_batch` in this file. I bet it already provides what you need.